docs/intro/ff-ui/builder.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: builder
+title: App Builder
+tags: []
+description: Explore the App Builder in FlutterFlow, featuring a comprehensive interface with four main sections. Navigation Menu, Toolbar, Canvas, and Properties Panel.
+---
+
+# App Builder
```

Comment:
Overall comment: Can we fix the sidebar positions of the pages and drop them in order? Use front matter to modify positioning.